lib/jekyll-last-modified-at/determinator.rb

```diff
@@ -30,7 +31,7 @@ def formatted_last_modified_date
       def last_modified_at_time
         raise Errno::ENOENT, "#{absolute_path_to_article} does not exist!" unless File.exist? absolute_path_to_article
 
-        Time.at(last_modified_at_unix.to_i)
+        @last_mofified_at_time ||= Time.at(last_modified_at_unix.to_i)
```

Comment:
I assume `mofified` is a typo. Did you test this?

Reply:
Yes, this is a typo. But it seems that it is just caching and does not affect correctness.